vitest.config.ts

```diff
@@ -33,14 +33,11 @@ export default defineConfig({
         'src/**/*.types.ts',
         'src/**/*.test.tsx',
         'src/**/*.stories.tsx',
+        'src/molecules/ListItem/ListItem.utils.tsx',
```

Comment:
Any reason this file is being ignored for coverage?

Reply:
It is not anymore 👌